### PR TITLE
API Design

### PR DESCRIPTION
**Pull request by ashishkava**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+
+// assume for now this is your database
+const swaggerUi = require("swagger-ui-express"),
+    swaggerDocument = require("./swagger.json");
+
```
Comment (jaimin45) [inline app.js:7]: please used yaml 
```
const swaggerYaml = require('yamljs');
const swaggerYamlDocument  = swaggerYaml.load('./swagger.yaml'); 


app.use(
  '/api-docs',swaggerUi.serve,swaggerUi.setup(swaggerYamlDocument)
);
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon server.js --exec babel-node --presets babel-preset-env",
+    "lint": "eslint src/js"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": { 
+    "body-parser": "^1.19.0",   
+    "express": "^4.17.1",
+    "nodemon": "^2.0.15",
```
Comment (jaimin45) [inline package.json:17]: please add  nodemon in dev dep
```
npm install nodemon --save -dev

```

**Review by jaimin45 — CHANGES_REQUESTED** [review]

File: swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,93 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Tech Holding 
+  description: An app to manage Tutorials
+host: localhost:8000
+basePath: "/"
+tags:
+- name: tutorials
```
Comment (JainishaParmar) [inline swagger.yaml:10]: name first letter always capital
```
name:Tutorial
```

File: swagger.yaml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  description: API for user in the system
+schemes:
+- http
+consumes:
+- application/yaml
+produces:
+- application/yaml
```
Comment (JainishaParmar) [inline swagger.yaml:17]: should be 
```
application/json
```

File: swagger.yaml [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+paths:
+  tutorial:
+    parameters:
+    - name: id
+      in: path
+      required: true
+      description: ID of tutorial that we want to find
+      type: integer
```
Comment (JainishaParmar) [inline swagger.yaml:25]: should be:
```
type: string
format: uuid
```

File: swagger.yaml [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+definitions:
+  tutorial:
+    required:
+    - UUID
+    - createdAt
+    - updatedAt
+    properties:
+      UUID:
+        type: String
+        uniqueItems: true
```
Comment (JainishaParmar) [inline swagger.yaml:85]: The name of this field should be _id but the type should be string and format should be uuid.

```
_id: 
  type: string
  format: uuid
```

**Review by JainishaParmar — CHANGES_REQUESTED** [review]

**Review by jaimin45 — APPROVED** [review]

**Pull request closed without merging**